README.md

```diff
@@ -5,40 +5,46 @@ Java utility for uploading code to Fortify on Demand
+ -z			     | -zipLocation                  | Yes               | Location of payload zip file
+ -ep		     | -entitlementPreferenceType    | Yes               | Whether to use a single scan or subscription assessment: 1/SingleScanOnly, 2/SubscriptionOnly, 3/SingleScanFirstThenSubscription, 4/SubscriptionFirstThenSingleScan 
+ -ac		     | -apiCredentials               | Yes<sup>1</sup>   | API credentials credentials                                                 
+ -uc		     | -userCredentials              | Yes<sup>1</sup>   | User credentials (wrap each in quotations to handle certain non-alphanumeric characters in the CLI) 
+ -bsi	    	| -bsiToken                     | Yes<sup>2</sup>   | BSI token
+ -rid	    	| -releaseId                    | Yes<sup>2</sup>   | Release ID
+ -purl		   | -portalurl                    | Yes<sup>3</sup>   | Domain URL
+ -aurl		   | -apiurl                       | Yes<sup>3</sup>   | API root URL
+ -tc		     | -tenantCode                   | Yes<sup>3</sup>   | Tenant ID if using user credentials 
+ -at		     | -assessmentTypeId				         | Yes<sup>4</sup>   | Assessment type ID
+ -eid	    	| -entitlement					             | Yes<sup>4</sup>   | Entitlement ID
+ -ts		     | -technologyStackId 			        | Yes<sup>4</sup>   | Technology stack as an integer: 1 (.NET), 23 (.Net Core), 2 (ABAP), 21 (Apex/Visualforce), 3 (ASP), 5 (CFML), 6 (COBOL), 22 (Go), 7 (JAVA/J2EE), 16 (JS/TS/HTML), 18 (MBS/C/C++/Scala), 9 (PHP), 10 (PYTHON), 17 (Ruby), 12 (Swift/Objective C/C++), 11 (VB6), 14 (VBScript)
+ -l			     | -languageLevelId				          | Yes<sup>4</sup>   | Language level as an integer:<br />.NET: 2 (2.0), 3 (3.0), 4 (3.5), 5 (4.0), 11 (4.5), 15 (4.6), 16 (4.7), 30, (4.8), 32 (5.0)<br /> .NET Core: 23 (1.0), 24 (1.1), 25 (2.0), 26 (2.1), 27 (2.2), 28 (3.0), 29 (3.1)<br />Java: 8 (1.5), 9 (1.6), 10 (1.7), 12 (1.8), 17 (1.9), 19 (10), 20 (11), 21 (12), 22 (13)<br /> Python: 13 (2), 14 (2 Django), 18 (3)
+ -a			     | -auditPreferenceId 		        	| Yes<sup>4</sup>	  | Audit preference: Manual, Automated
+ -bs		     | -isBinaryScan					            | No<sup>4</sup>	   | Scan compiled and source code (the feature must be enabled)
+ -os 		    | -allowopenSourceComponentAnalysis| No<sup>4</sup>	| Include open source component analysis
+ -rp       | -remediationScanPreferenceType| No                | Remediation scan preference: 0/RemediationScanIfAvailable, 1/RemediationScanOnly, 2/NonRemediationScanOnly (default)
+ -pp		     | -inProgressScanActionType     | No                | If in-progress scan exists, the action to take for a new scan: 0/DoNotStartScan (default), 1/CancelScanInProgress, 2/Queue
+ -purchase	| -purchaseEntitlement          | No				            | Whether to purchase an entitlement (if available)
+ -apf		| -allowPolicyFail                  | No                | Whether to return exit(0) instead of exit(1) if the scan fails the security policy specified in Fortify on Demand 
```

Comment:
The description was a bit unclear for the `-apf` flag. We finally understood that adding `-apf` will always exit 0 regardless of scan results and ignore the policy.